src/components/transactions/message-view/MultiSendMessageDetails.vue

```diff
@@ -60,12 +55,35 @@ export default {
     }
   },
   computed: {
+    recipient() {
+      let index = this.transaction.value.inputs.findIndex(
+        input => input.address == this.sessionAddress
+      )
+      if (index !== undefined && this.transaction.value.outputs[index]) {
+        return this.transaction.value.outputs[index].address
+      }
+      return false
+    },
+    sender() {
```

Comment:
same here, why is the sender necessarily in the outputs?

Reply:
Only my thoughts... Like if the user is in inputs (e.g. senders) he is a sender, if he is in outputs (e.g. recipients) he is a recipient

Reply:
Isn't it actually the opposite?

Reply:
True, my fault, but the code is right) We are looking for a sender if our address is a recipient and we are looking for a recipient if our address is sender.

Reply:
I get it. Deserves comments though